docs/source/getting_started/amd-installation.rst

```diff
@@ -0,0 +1,80 @@
+.. _installation:
+
+Installation with ROCm
+============
+
+vLLM 0.2.x onwards supports model inferencing and serving on AMD GPUs with ROCm.
+At the moment AWQ quantization is not supported in ROCm, but SqueezeLLM quantization has been ported.
+Datatypes currently supported in ROCm are FP16 and BF16.
+
+Requirements
+------------
+
+* OS: Linux
+* Python: 3.8 -- 3.11 (Verified on 3.10)
+* GPU: MI200s
+* Pytorch 2.0.1/2.1.1
+* ROCm >= 5.7.0
+
+
+.. _build_from_source:
+
+Build from source with docker
+-----------------
+
+You can also build and install vLLM from source:
+
+Build a docker image from `Dockerfile.rocm`, and launch a docker container.
+
+.. code-block:: console
+
+    $ docker build -f Dockerfile.rocm -t vllm-rocm . 
+    $ docker run -it \
+       --network=host \
+       --group-add=video \
+       --ipc=host \
+       --cap-add=SYS_PTRACE \
+       --security-opt seccomp=unconfined \
+       --shm-size 8G \
+       --device /dev/kfd \
+       --device /dev/dri \
+       -v <path/to/model>:/app/model \
+       vllm-rocm \
+       bash
+
```

Comment:
There is an additional step that I adapted from the v0.2.1.post1-rocm branch, which is to run the run_benchmark.sh script available in that branch (it need to run "mkdir dataset" in /app directory before running the script.
It might be worth mentioning here after start container step.

Reply:
We are planning to incorporate the steps from v0.2.1.post1-rocm branch into the [documentations](https://docs.vllm.ai/en/latest/) in the coming PRs.
@WoosukKwon What do you think?